pages/en/lb3/Storage-component.md

```diff
@@ -300,6 +300,39 @@ as shown in the following table.
 
 As documented above, a file uploaded with the same name as an existing file will be overwritten. By adding the configuration key `nameConflict` with the value `makeUnique`, files will automatically be renamed with a UUID and the existing file extension of the original file name. Most likely this is an option that you will want to enable, otherwise you will need to ensure uniqueness on the code calling the API.
 
+## Renaming files
+
+You can rename uploading files before they reach their destination - file system or cloud - by setting the `getFilename` function in its datasource options. This can be done either in a model file or in a boot script, or anywhere you can access the datasource object.
```

Comment:
Suggestion:
You can rename a files being uploaded before they reach ...

Reply:
Or maybe simply, "You can rename files before they reach their destination...".